tango_ros_common/tango_ros_messages/srv/SaveMap.srv

```diff
@@ -1,6 +1,11 @@
+int8 LOC_AND_NAV_MAPS=0
+int8 ONLY_LOC_MAP=1
```

Comment:
It would make sense to encode this binary:
LOAD_LOCALIZATION_MAP = 1
LOAD_NAVIGATION_MAP/ LOAD_OCCUPANCY_GRID = 2

Loading both would be possible as:
LOAD_OCCUPANCY_GRID | LOAD_LOCALIZATION_MAP